terps/scott/TI99_4a_terp.c

```diff
@@ -0,0 +1,570 @@
+#include <string.h>
+
+#include "glk.h"
+#include "load_TI99_4a.h"
+#include "scott.h"
+#include "TI99_4a_terp.h"
+
+ActionResultType PerformTI99Line(uint8_t *action_line)
+{
+    if (action_line == NULL)
+        return 1;

```

Comment:
Since this returns an enum, it ought to use the enum names (ACT_FAILURE here, preusmably).

Reply:
Done!